src/web/templates/edit_queue.html

```diff
@@ -1,6 +1,11 @@
 {% extends "layout.html" %}
 {% block content %}
-<div><h2>{{ queue.name|e }} (ID {{ queue.id }})</h2></div>
+<div style="margin: 5px">
+    <input type="text" id="new_queue_name" class="new_queue_name" size="30" placeholder="{{ queue.name|e }}">
```

Comment:
Instead of using `placeholder`, let's initialize the text contents of the input to the current name

Reply:
I dont really think it is comfortable.